package.json

```diff
@@ -71,6 +72,7 @@
     "husky": "^8.0.3",
     "jest": "^29.4.1",
     "jest-environment-jsdom": "^29.4.1",
+    "jest-environment-node": "^29.4.1",
```

Comment:
please remove this package, as we don't need it and it's causing the CI/CD pipeline to fail.